src/gateways/PrismaGouvernanceRepository.test.ts

```diff
@@ -81,12 +82,89 @@ describe('gouvernance repository', () => {
     await prisma.noteDeContexteRecord.create({
       data: noteDeContexteRecordFactory({ gouvernanceDepartementCode: departementCode }),
     })
-    const repository = new PrismaGouvernanceRepository(prisma.gouvernanceRecord)
+    const repository = new PrismaGouvernanceRepository(prisma.gouvernanceRecord, prisma.noteDeContexteRecord)
 
     // WHEN
     const gouvernanceTrouvee = await repository.find(new GouvernanceUid(departementCode))
 
     // THEN
     expect(gouvernanceTrouvee?.state).toStrictEqual(gouvernanceFactory({ uid: '75' }).state)
   })
+
+  it('ajouter une note de contexte à une gouvernance', async () => {
+    // GIVEN
+    const gouvernanceId = 1
+    await prisma.regionRecord.create({
+      data: regionRecordFactory({ code: '11' }),
+    })
+    await prisma.departementRecord.create({
+      data: departementRecordFactory({ code: '75' }),
+    })
+    await prisma.utilisateurRecord.create({
+      data: utilisateurRecordFactory({ id: 1, ssoId: 'userFooId' }),
+    })
+
+    await prisma.gouvernanceRecord.create({
+      data: gouvernanceRecordFactory({ departementCode: '75' }),
+    })
+
+    const repository = new PrismaGouvernanceRepository(prisma.gouvernanceRecord, prisma.noteDeContexteRecord)
+    const gouvernanceMiseAJourAvecNoteDeContexte = gouvernanceFactory({
+      noteDeContexte: {
+        contenu: '<p>lorem ipsum dolor sit amet</p>',
+        dateDeModification: new Date(),
+        uidEditeur: new UtilisateurUid({
+          email: 'martin.tartempion@example.net',
+          value: 'userFooId',
+        }),
+      },
+      uid: String(gouvernanceId),
```

Comment:
```suggestion
      uid: '75'
```
Il n'y a plus d'`id` de gouvernance, le code département fait office de clef primaire. Comment ce test peut-il passer alors qu'on met à jour une gouvernance qui n'existe pas ?